sidecar-injector/README.md

```diff
@@ -126,61 +267,346 @@ For this section, you'll work from a test namespace (test-namespace). Later you
+    cyberark-sidecar-injector       Active    18h
+    conjur-sidecar-test             Active    18h       enabled
+    ```
+
+1. Create service account (might already exist from `kubernetes-conjur-demo`) to be used by the application pod
     
-    ~$ kubectl \
-      -n test-namespace \
-      exec ${a_sleep_pod} \
-      -c sleep \
+    This service account maps to the Conjur identity for the pod
+
+    ```bash
+    ~$ kubectl -n ${TEST_APP_NAMESPACE_NAME} \
+    create serviceaccount ${TEST_APP_SERVICE_ACCOUNT}
+    ```
+
+1. Create Conjur ConfigMap
+
+    This ConfigMap named `conjur` stores the connection details to the Conjur appliance.
+    These details are necessary for both the **Authenticator** and **Secretless** sidecars to communicate with the Conjur appliance.
+    ```bash
+    ~$ cat << EOL | kubectl -n ${TEST_APP_NAMESPACE_NAME} apply -f -
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: conjur
+    data:
+      CONJUR_ACCOUNT: "${CONJUR_ACCOUNT}"
+      CONJUR_VERSION: "${CONJUR_VERSION}"
+      CONJUR_APPLIANCE_URL: "${CONJUR_APPLIANCE_URL}"
+      CONJUR_AUTHN_URL: "${CONJUR_AUTHN_URL}"
+      CONJUR_SSL_CERTIFICATE: |
+    $(echo "${CONJUR_SSL_CERTIFICATE}" | awk '{ print "    " $0 }')
+      CONJUR_AUTHN_LOGIN: "${CONJUR_AUTHN_LOGIN}"
+    EOL
+    ```
+
+1. You can now leverage Conjur by either
+   1. [Deploying the Authenticator Sidecar](#deploy-authenticator-sidecar) or
+   1. [Deploying the Secretless Sidecar](#deploy-authenticator-sidecar) 
+
+### Deploy Authenticator Sidecar
+
+1. Deploy an app with the Authenticator Sidecar:
+    
+    The **Secretless Sidecar** is injected into the application pod on pod creation via the sidecar injector. The injection is configured via annotations. 
+    
+    + The `conjur` ConfigMap is used for both Conjur Authentication and Connection configuration
+    + The `sidecar-injector.cyberark.com/containerName` is set to "secretless" because the corresponding Conjur identity to the service account used expects the sidecar container to be named secretless.
+
+    ```bash
+    ~$ kubectl -n ${TEST_APP_NAMESPACE_NAME} \
+      delete pod \
+      test-app --ignore-not-found
+   
+    ~$ cat << EOF | kubectl -n ${TEST_APP_NAMESPACE_NAME} apply -f -
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      annotations:
+        sidecar-injector.cyberark.com/conjurAuthConfig: conjur
+        sidecar-injector.cyberark.com/conjurConnConfig: conjur
+        sidecar-injector.cyberark.com/containerMode: ${containerMode}
+        sidecar-injector.cyberark.com/inject: "yes"
+        sidecar-injector.cyberark.com/injectType: authenticator
+        sidecar-injector.cyberark.com/containerName: secretless
+      labels:
+        app: test-app
+      name: test-app
+    spec:
+      containers:
+      - image: googlecontainer/echoserver:1.1
+        name: app
+        volumeMounts:
+        - mountPath: /run/conjur
+          name: conjur-access-token
```

Comment:
Getting this error when trying to run this command:
```
The Pod "test-app" is invalid: spec.containers[0].volumeMounts[0].name: Not found: "conjur-access-token"
```

Reply:
I think you need to define the volume, something like:
```
  volumes:
    - name: conjur-access-token
      emptyDir:
        medium: Memory
```

Reply:
This volume isn't necessary and should be added by the sidecar injector

Reply:
@jtuttle I think the reason it's not working is because there are no injectors in GKE. i'd purged everything since yesterday you'd have to start from scratch and install an injector then run the examples